src/PluginsRegistry.js

```diff
@@ -6,46 +6,59 @@
  * @api public
  */
 export default class PluginsRegistry {
-  registry: Array<Function>;
+  registry: Array<Object>
+  jss: Object
 
-  constructor(): void {
+  constructor(jss: Object): void {
     this.registry = []
+    this.jss = jss
```

Comment:
Oh, I could actually pass jss as a second argument to the .use method. This would work for now, not sure though this would work forever, next time it is needed in another method its not any more nice. Also kinda not very nice for the .use method to require this second argument additionally to the plugin. 

I agree that having the jss instance all over the place is not very nice, but I don't see a real good another way. Also which problems can having .jss property bring? Harder to unit test?